Framework/API/src/PreviewDeclarations.cpp

```diff
@@ -0,0 +1,197 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
This entire file can be safely removed, it is not used in the current version of the RDExp.